packages/manager/apps/pci-cold-archive/src/pages/containers/archive/Archive.page.tsx

```diff
@@ -89,9 +89,11 @@ export default function ArchivePage() {
 
   const onClose = onCancel;
 
+  const maxDayRetention = 4500;
```

Comment:
As a minor improvement

```suggestion
  const maxDaysRetention = 4500;
```